### PR TITLE
Updated scenario at flex zone for FFI demo

### PR DESCRIPTION
**Pull request by LukasWikander**

### CODE REVIEW

File: scripts/openscenario/ffi_scenario.py [hunk 1]
```diff
@@ -33,49 +33,49 @@
 mondeo_slow_speed = 10 / 3.6 # m/s
 mondeo_acceleration = 0.51 # m/s^2
 mondeo_retardation = 2.17 # m/s^2
-mondeo_O1_point = xosc.LanePosition(83.5103988647461, 0, -1, 3)
+mondeo_O1_point = xosc.LanePosition(15, 0, -1, 2)
```
Comment (victorjarlow) [inline scripts/openscenario/ffi_scenario.py:36]: might improve readability to call it mondeo_starting_point

Comment (victorjarlow) [reply scripts/openscenario/ffi_scenario.py:36]: Also creating variables for the lanes / roads e.g. LEFT_LANE = -1

**Review by victorjarlow — APPROVED** [review]

File: scripts/openscenario/ffi_scenario.py (hunk 1 above)
Comment (LukasWikander) [reply scripts/openscenario/ffi_scenario.py:36]: The lane IDs are not consistent since it is not fixed in what "direction" a road is pointing

Comment (victorjarlow) [reply scripts/openscenario/ffi_scenario.py:36]: Ok I see, too bad!

**Pull request merged**